test/system/src/test/java/io/pravega/test/system/BookieFailoverTest.java

```diff
@@ -51,7 +51,7 @@
     private static final String SCOPE = "testBookieFailoverScope" + RandomFactory.create().nextInt(Integer.MAX_VALUE);
     private static final int NUM_WRITERS = 5;
     private static final int NUM_READERS = 5;
-    private static final int BOOKIE_FAILOVER_WAIT_MILLIS = 20 * 1000;
+    private static final int BOOKIE_FAILOVER_WAIT_MILLIS = 15 * 1000;
```

Comment:
Is this change related with this PR?

Reply:
Yes, as the the wait time is stalling the test for a longer time due to which the BookieFailoverTest is failing for operator `0.4.0`

Reply:
I do not understand why this sleep makes the test to fail with a different operator version. Could you please provide more details here?

Reply:
In the BookieFailover test, after bringing down  one bookie replica, we wait for twice the wait time before a new bookie instance is spawned again. In the meanwhile neither writes or reads will be performed. When the new bookie instance comes up, it again takes some time before the Segment Store is able to connect to this bookie and the reads and writes are able to resume only after this. If the wait time is kept long, the test times out before the reads and writes are able to complete.

Reply:
This does not answer my question. This test has been working for a long time without problems. Why changing the operator version requires changing this timeout? If this change is not required, then is better to remove it; if the change is actually needed, we need at least to understand why with the new operator this test requires a shorter timeout.

Reply:
An `InitialDelay` was added to the `Readiness Probe` of the bookie after `0.3.0` as can be seen here : https://github.com/pravega/pravega-operator/commit/3280f8b07609d18d1d2f5f834edbd5cbecf21001#diff-4620fbe56580da27f4724ef7207b8d07R126
This might be reason that bookie pods take a few extra seconds with operator `0.4.0`